services/federation/follow_calls.go

```diff
@@ -0,0 +1,51 @@
+package federation
+
+import (
+	"github.com/yitsushi/go-misskey/core"
+	"github.com/yitsushi/go-misskey/models"
+)
+
+// FollowRequest is the request used by the followers and following commands.
+type FollowRequest struct {
+	Host    string `json:"host"`
+	SinceID string `json:"sinceId"`
+	UntilID string `json:"untilId"`
+	Limit   int    `json:"limit"`
+}
+
+// Follow is a single follower or following record.
+type Follow struct {
+	ID         string      `json:"id"`
+	CreatedAt  string      `json:"createdAt"`
+	FolloweeID string      `json:"followeeId"`
+	Followee   models.User `json:"followee"`
+	FollowerID string      `json:"followerId"`
+	Follower   models.User `json:"follower"`
```

Comment:
Random note: This key is not even in the response for `/federation/followers` or `/federation/following`.

Reply:
Are we looking at the same documentation? I'm looking at the one in the issue. Please check that.

Reply:
```json
[
{
"id": "xxxxxxxxxx",
"createdAt": "2019-08-24T14:15:22Z",
"followeeId": "string",
"followee": {...},
"followerId": "string",
"follower": {...}
}
]
```

Reply:
I was looking at the endpoint itself XD

```
curl -X POST -d '{"host": "misskey.io"}' https://slippy.xyz/api/federation/following | jq
curl -X POST -d '{"host": "misskey.io"}' https://slippy.xyz/api/federation/followers | jq
```

Leave the field there, as it may be used later or elsewhere (like `/following/*`)

Reply:
That's why I have comments like these:
 - https://github.com/yitsushi/go-misskey/pull/29/files#diff-e0f4566ff139c41b6f558e65f4d84eaea53e9028298f77c53f2d8862266cd028R11
 - https://github.com/yitsushi/go-misskey/pull/29/files#diff-0379afa1d1bd663f6ea069326a7bf6d5824d157f3338895b991bcbf2601418c5R31